test/actions/submit-drake.js

```diff
@@ -162,6 +158,41 @@ describe('submitDrake action', () => {
 
   });
 
+  describe('when the drake is submitted as an offspring', () => {
+    const correct = true,
+          dispatch = expect.createSpy();
+
+    actions.submitDrake(0, 1, correct, null, 2, 3)(dispatch, getState);
+
+    it('should call dispatch with the correct _submitDrake action', () => {
+      var submitDrakeAction = dispatch.calls[0].arguments[0];
+      expect(submitDrakeAction).toEqual({
+        type: types.DRAKE_SUBMITTED,
+        species: "Drake",
+        challengeCriteria: {
+          phenotype: submitDrakeAction.challengeCriteria.phenotype,  // we check valid phenotype below
```

Comment:
Tiny thing - the comment about phenotype-checking comes from the call to `assertMatchingPhenotype` in other tests, which isn't included here. Since the point of the test is really to check that the offspring pieces of this action are correct, the assertion might be unnecessary, but the comment should probably reflect that.